packages/dashboards/addon/components/dashboard-header.hbs

```diff
@@ -3,7 +3,7 @@
   <div class="row">
     <div class="col-1-1">
       {{#if (feature-flag "enableDirectory")}}
-        <LinkTo @route="directory.my-data" class="dashboard-header__breadcrumb-link">Directory</LinkTo>
+        <LinkTo @route="index" class="dashboard-header__breadcrumb-link">Home</LinkTo>
       {{else}}
         <LinkTo @route="dashboards" class="dashboard-header__breadcrumb-link">Dashboards</LinkTo>
       {{/if}}
```

Comment:
We should be able to remove the `#if` and just route to index in both situations